packages/image_picker/image_picker_android/README.md

```diff
@@ -33,5 +33,8 @@ import 'package:image_picker_platform_interface/image_picker_platform_interface.
   }
 ```
 
+In addition, `[ImagePickerAndroid.useAndroidPhotoPicker]` must be set to `true` to use the `limit` functionality. It is implemented based on [`ActivityResultContract`][3], so it can only be ensured to take effect on Android 13 or above. Otherwise, it depends on whether the corresponding system app supports it.
```

Comment:
Nit: `[ImagePickerAndroid.useAndroidPhotoPicker]` shouldn't have brackets, since this isn't a Dartdoc comment.

Reply:
Thx.